src/tests/ftest/soak/soak_test_base.py

```diff
@@ -504,24 +510,26 @@
         self.dmg_command.copy_configuration(self.hostlist_clients)
         harassers = self.params.get("harasserlist", test_param + "*")
         job_list = self.params.get("joblist", test_param + "*")
-        rank = self.params.get("rank", "/run/container_reserved/*")
-        obj_class = self.params.get("oclass", "/run/container_reserved/*")
         if harassers:
             run_harasser = True
-            self.log.info("<< Initial harasser list = %s>>", " ".join(
-                [harasser for harasser in harassers]))
+            self.log.info("<< Initial harasser list = %s>>", harassers)
             harasserlist = harassers[:]
         # Create the reserved pool with data
         # self.pool is a list of all the pools used in soak
         # self.pool[0] will always be the reserved pool
         add_pools(self, ["pool_reserved"])
-        self.pool[0].connect()
-
-        # Create the container and populate with a known data
-        # TO-DO: use IOR to write and later read verify the data
+        # Create the reserved container
         resv_cont = self.get_container(
             self.pool[0], "/run/container_reserved/*", True)
-        resv_cont.write_objects(rank, obj_class)
+        # populate reserved container with a 500MB file
+        initial_resv_file = os.path.join(
+            os.environ["DAOS_TEST_LOG_DIR"], "initial", "resv_file")
+        try:
+            reserved_file_copy(self, initial_resv_file, self.pool[0], resv_cont,
+                      num_bytes=500000000, cmd="write")
+        except CommandFailure as error:
+            raise SoakTestError(
```

Comment:
(pylint-raise-missing-from) Consider explicitly re-raising using the 'from' keyword